versioneer.py

```diff
@@ -620,7 +620,7 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # refs/heads/ and refs/tags/ prefixes that would let us distinguish
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
-        # "stabilization", as well as "HEAD" and "master".
+        # "stabilization", as well as "HEAD" and "main".
```

Comment:
This is a comment made within the versioneer code. No harm in changing but I think it would invalidate the comment.

Reply:
I think the comment is still valid.

Reply:
Oh yeah I guess so!